src/Tooltip/Tooltip.component.tsx

```diff
@@ -21,6 +18,9 @@ export class Tooltip extends React.Component<Props> {
     arrow: true,
     theme: Themes.defaultTheme,
   };
+  /**
+   * What the reason of this function?
+   */
```

Comment:
I don't think we should leave this here. Seems like its not being used. Git Lens says @ben-dalton had added it 4 months when setting up the component.

Reply:
Removed